cabal-install/Distribution/Client/Setup.hs

```diff
@@ -697,6 +701,7 @@ defaultFreezeFlags = FreezeFlags {
     freezeSolver           = Flag defaultSolver,
     freezeMaxBackjumps     = Flag defaultMaxBackjumps,
     freezeReorderGoals     = Flag (ReorderGoals False),
+    freezeCountConflicts   = Flag (CountConflicts False),
```

Comment:
Shouldn't `fetch` and `freeze` use the same default solver parameters as `install`?